libdeno/binding.cc

```diff
@@ -389,6 +483,18 @@ void InitializeContext(v8::Isolate* isolate, v8::Local<v8::Context> context) {
   auto send_val = send_tmpl->GetFunction(context).ToLocalChecked();
   CHECK(deno_val->Set(context, deno::v8_str("send"), send_val).FromJust());
 
+  auto eval_context_tmpl = v8::FunctionTemplate::New(isolate, EvalContext);
+  auto eval_context_val =
+      eval_context_tmpl->GetFunction(context).ToLocalChecked();
+  CHECK(deno_val->Set(context, deno::v8_str("evalContext"), eval_context_val)
+            .FromJust());
+
+  auto error_to_json_tmpl = v8::FunctionTemplate::New(isolate, ErrorToJSON);
+  auto error_to_json_val =
+      error_to_json_tmpl->GetFunction(context).ToLocalChecked();
+  CHECK(deno_val->Set(context, deno::v8_str("errorToJSON"), error_to_json_val)
+            .FromJust());
+
```

Comment:
Hmm. I still wonder if there isn’t a better way. 

Let’s just go with it for now tho.

Reply:
Could not find a better way at this moment, since there must need to have a step that transforms an `Error` object into `v8_exception` JSON format that Rust side can understand...

Actually, `libdeno.errorToJSON()` might be a useful utility for third-party modules to build their custom error formatting and diagnosis...